modelerfour/modeler/modelerfour.ts

```diff
@@ -1228,6 +1220,16 @@ export class ModelerFour {
   }
 
   process() {
+    let priority = 0;
```

Comment:
Could help to remember what this is for later with a name like `globalParameterPriority`